rbac/management/group/view.py

```diff
@@ -97,6 +97,16 @@ def get_serializer_class(self):
             return GroupInputSerializer
         return GroupSerializer
 
+    def protect_default_groups(self, group, action):
+        """Deny modifications on platform_default groups."""
+        if group.platform_default:
+            key = 'group'
+            message = '{} cannot be performed on platform default groups.'.format(action)
```

Comment:
One minor suggestion: the responses might read better if we uppercase the action?
```suggestion
            message = '{} cannot be performed on platform default groups.'.format(action.upper())
```

```
{
  "errors": [
    {
      "detail": "UPDATE cannot be performed on platform default groups.",
      "source": "group",
      "status": 400
    }
  ]
}
```